include/adapter/parameters.cc

```diff
@@ -0,0 +1,198 @@
+#include <adapter/parameters.h>
+
+namespace Parameters
+{
+  void
+  Time::add_output_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("Time");
+    {
+      prm.add_parameter("End time", end_time, "End time", Patterns::Double());
+
+      prm.add_parameter("Time step size",
+                        delta_t,
+                        "Time step size",
+                        Patterns::Double());
+
+      prm.add_parameter("Output interval",
+                        output_interval,
+                        "Write results every x timesteps",
+                        Patterns::Integer(0));
+    }
+    prm.leave_subsection();
+  }
+
+
+  void
+  System::add_output_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("System properties");
+    {
+      prm.add_parameter("Shear modulus",
+                        mu,
+                        "Shear modulus",
+                        Patterns::Double());
+
+      prm.add_parameter("Poisson's ratio",
+                        nu,
+                        "Poisson's ratio",
+                        Patterns::Double(-1.0, 0.5));
+
+      prm.add_parameter("rho", rho, "density", Patterns::Double(0.0));
+
+      prm.add_parameter("body forces",
+                        body_force,
+                        "body forces x,y,z",
+                        Patterns::List(Patterns::Double()));
+    }
+    prm.leave_subsection();
+
+    lambda = 2 * mu * nu / (1 - 2 * nu);
+  }
+
+
+  void
+  Solver::add_output_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("Solver");
+    {
+      prm.add_parameter("Model",
+                        model,
+                        "Structural model to be used: linear or neo-Hook",
+                        Patterns::Selection("linear|neo-Hook"));
```

Comment:
Done. I added now an option here, where the user can select between the linear model and the neo-Hookean model. I hope this is intuitive enough, otherwise let me know.